# place the stencil plane in red instead of its specified green component.
diff --git a/src/winemetal/winemetal.h b/src/winemetal/winemetal.h
--- a/src/winemetal/winemetal.h
+++ b/src/winemetal/winemetal.h
@@ -455,7 +455,7 @@ enum WMTPixelFormat {
   WMTPixelFormatBGRX8Unorm_sRGB = WMTPixelFormatAlphaIsOne | WMTPixelFormatBGRA8Unorm_sRGB,
 
   WMTPixelFormatR32X8X32 = WMTPixelFormatR001Swizzle | WMTPixelFormatDepth32Float_Stencil8,
-  WMTPixelFormatX32G8X32 = WMTPixelFormat0R01Swizzle | WMTPixelFormatX32_Stencil8,
+  WMTPixelFormatX32G8X32 = WMTPixelFormatR001Swizzle | WMTPixelFormatX32_Stencil8,
 
   WMTPixelFormatBGRA4Unorm = WMTPixelFormatGBARSwizzle | WMTPixelFormatABGR4Unorm,
 };
